library/core/src/test/java/com/google/android/exoplayer2/trackselection/DefaultTrackSelectorTest.java

```diff
@@ -1078,6 +1078,33 @@ public void testSelectTracksWithinCapabilitiesAndForceLowestBitrateSelectLowerBi
     assertThat(result.selections.get(0).getSelectedFormat()).isEqualTo(lowerBitrateFormat);
   }
 
+  /**
+   * Tests that track selector will select audio tracks with higher bitrate when {@link Parameters}
+   * indicate highest bitrate preference, even when tracks are within capabilities.
+   */
+  @Test
+  public void testSelectTracksWithinCapabilitiesAndForceHighestBitrateSelectHigherBitrate()
+          throws Exception {
+    trackSelector.setParameters(new ParametersBuilder().setForceHighestSupportedBitrate(true).build());
+
+    Format lowerBitrateFormat =
+            Format.createAudioSampleFormat("audioFormat", MimeTypes.AUDIO_AAC, null, 15000,
+                    Format.NO_VALUE, 2, 44100, null, null, 0, null);
+    Format higherBitrateFormat =
```

Comment:
Would be nice to add a third format (with an even higher bitrate) which exceeds the renderer capabilities. See for example testSelectTracksPreferTrackWithinCapabilities for how to configure that.